### PR TITLE
Basic game phase

### PR DESCRIPTION
**Pull request by richard1**
Implementation of a global static game phase.  Rough prototype, open to suggestions.

### CODE REVIEW

**Review by AznCelebration — APPROVED** [review]

**Review by metakirby5 — APPROVED** [review]
0/10 not in chef

File: src/game/phase.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "../events.h"
```
Comment (metakirby5) [inline src/game/phase.h]: Don't need `../`

**Comment by richard1** [comment]
@metakirby5 ty for review, but planning to close this PR.  Kavin and I are working on the `build-integration` branch which has this branch's changes + some slight improvements, so we'll submit a PR from there when that's ready.

**Comment by metakirby5** [comment]
@richard1 Would it not be better to do incremental reviews, so we are bombarded with less code to read?

**Comment by richard1** [comment]
@metakirby5 

U R RITE

**Pull request merged**